django_extras/core/types.py

```diff
@@ -57,7 +57,7 @@ def decimal_value(value):
     return value
 
 
-class Money(object):
+class Money(decimal.Decimal):
```

Comment:
This is not the right solution to this problem. This class composes a money value from a Decimal amount and a currency it does not use inheritance.